src/ImageSharp/Processing/Processors/Convolution/Parameters/BokehBlurExecutionMode.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Processing.Processors.Convolution.Parameters
```

Comment:
Move this to `SixLabors.ImageSharp.Processing` please so it's easier for devs to consume. We lump all options there.

Reply:
Done in https://github.com/SixLabors/ImageSharp/pull/842/commits/9aa75ccf4eae64a8be5be0f3a1cdb6d625a46da8.